README.md

```diff
@@ -180,6 +180,19 @@ dnf install cosmic-desktop
 Then log out, click on your user, and a sprocket at the bottom right shows an additional entry alongside your desktop environments. Change to COSMIC and proceed with log in.
 For further information, you may check the [COPR page](https://copr.fedorainfracloud.org/coprs/ryanabx/cosmic-epoch/).
 
+## Installing on openSUSE tumbleweed 
+Cosmic can be installed by adding rrahl0 repo with opi.
+```
+opi xdg-desktop-portal-cosmic
```

Comment:
Shouldn't it be `cosmic-session`?

Reply:
Installing `cosmic-session` does not install all needed packages, It will simply add the option to launch Cosmic, if selected the user will get kicked back to the login screen.